src/testing/deploy/deploy_test.go

```diff
@@ -40,7 +43,7 @@ func TestInstallAndUpgradeEnterpriseWithEnv(t *testing.T) {
 	c.SetAuthToken("")
 	mockIDPLogin(t, c)
 	// Test Upgrade
-	opts.CleanupAfter = true
+	opts.CleanupAfter = false
```

Comment:
ive seen this get toggled a lot. should we clean up after test?

Reply:
I reverted it because I haven't tested enough, but I'm becoming increasingly convinced that `cleanupAfter=true` doesn't make a functional difference and really only makes the pipelines harder to debug because the `dump k8s info` step can't dump the cluster state.